realsense2_camera_srvs/srv/CalibrateImuReq.srv

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Why did you create a new service? will the service client be interested on the heading result? I don't see this would be the case

Reply:
I can only think of having a database of the calibrated angle as the case in which the client would be interested in that value. However, I could change the service to use a [trigger](https://github.com/ros2/common_interfaces/blob/rolling/std_srvs/srv/Trigger.srv) and if needed, send the calibrated angle in the string message field.

Reply:
I think the "database" node will not be the one requesting the calibration. However lets leave it this way until we create that "database" node